### ISSUE TITLE
`headerGroupClasses` is marked as hidden

### ISSUE THREAD

**Issue by ymita** [IgniteUI/igniteui-angular#4276]
## Description  
`headerGroupClasses` is marked as hidden so that our doc does not show it.

 * igniteui-angular version: 7.2.0
 * browser: n/a

## Steps to reproduce  

1. See the source code - https://github.com/IgniteUI/igniteui-angular/blob/master/projects/igniteui-angular/src/lib/grids/column.component.ts#L347-L351

## Result  
`headerGroupClasses` is marked as hidden.

## Expected result  
`headerGroupClasses` is not marked as hidden.

## Attachments  
Attach a sample if available, and screenshots, if applicable.

**Comment by mpavlinov** [comment]
@ymita What's the reasoning of this request? I believe this property is hidden on purpose, not by mistake.

**Comment by ymita** [comment]
@mpavlinov 
A customer would like to use this property for group header. Can we make this property available for customer developers?

**Comment by mpavlinov** [comment]
@ymita I see, but what is the use case he's trying to achieve? Is it a styling issue that we don't cover with our theming functionality?
Maybe the solution for the customer is not opening up the private property. That's what I'm trying to understand.

**Comment by zdrawku** [comment]
I believe that the customer wants to align the text of group column header to center.
Mihoko achieved this requirement by using `headerGroupClasses` input like so:

```
<igx-column-group ... headerGroupClasses="app-group-column-header">
....
</igx-column-group>
```

```
.app-group-column-header > div.igx-grid__thead-title {
      justify-content: center !important;
}
```

If we want to do it in "the right way", we may consider creating of a `multi-header-text-align` [property](https://www.infragistics.com/products/ignite-ui-angular/docs/sass/latest/#variable-_dark-grid) that can help us to achieve this by using [grid schema](https://www.infragistics.com/products/ignite-ui-angular/angular/components/themes/schemas.html)

@ymita, using of schemas is the official and declarative way to define all properties a component theme might use.

@simeonoff, could you share your expert opinion on that matter?

**Comment by simeonoff** [comment]
@zdrawku If the target is to have a custom headers, different from the default ones, I'd suggest using custom header templates.

**Comment by zdrawku** [comment]
As we discussed with @SAndreeva, we are going to expose `headerGroupClasses`, the purpose of this input is to provide a class selector of the **Grouped headers**, such as `headerClasses` input which is used to get/set class selector of the **Column header**. 

Related [PR](https://github.com/IgniteUI/igniteui-angular/blob/master/projects/igniteui-angular/src/lib/grids/column.component.ts#L346)

**Comment by zdrawku** [comment]
`headerGroupClasses` is now available from the [API docs](https://www.infragistics.com/products/ignite-ui-angular/docs/7.1.10/typescript/classes/igxcolumngroupcomponent.html#headergroupclasses)

**Comment by ymita** [comment]
@zdrawku 
Thanks. Now I see `headerGroupClasses` is available on the 7.1.10 doc. Is this also available on the 7.2.1 doc as well?

**Comment by zdrawku** [comment]
The [PR ](https://github.com/IgniteUI/igniteui-angular/pull/4283) was merged 4 days ago, which means it will be available in 7.2.2 version which we will release today.
